scripts/apps/workspace/content/views/FormattingOptionsMultiSelect.tsx

```diff
@@ -1,36 +1,20 @@
 import React from 'react';
 import {TreeSelect} from 'superdesk-ui-framework/react';
-import {IArticleField} from 'superdesk-api';
+import {IArticleField, RICH_FORMATTING_OPTION} from 'superdesk-api';
 import {gettext} from 'core/utils';
-import {
-    getEditor3RichTextFormattingOptions,
-    getEditor3PlainTextFormattingOptions,
-    HAS_RICH_FORMATTING_OPTIONS,
-} from '../components/get-content-profiles-form-config';
 
 interface IProps {
     value: Array<string> | null;
     fieldId: string;
     fields: Dictionary<string, IArticleField>;
     onChange(value: Array<string>, fieldId: string): void;
+    formattingOptions: Array<{value: [RICH_FORMATTING_OPTION, string]}>;
```

Comment:
I agree, not only because of the one usage but because the available formattingOptions calculation doesn't happen inside the component and also my usage passes onChange with a different signature